build/run-e2e-tests.sh

```diff
@@ -78,6 +78,9 @@ npm run test:cypress-headless
 unset KUBECONFIG
 rm ${DIR}/tmpkube
 
+cp -r $DIR/../test-output/cypress/screenshots ${ARTIFACT_DIR}/${e2e-tests}/e2e/
```

Comment:
Why not just copy entire `test-output` folder?

Reply:
True--I was trying to be more targeted, but if this works I could just copy the whole folder over. There probably isn't sensitive information in the reports, anyway.